metadata/src/main/java/org/apache/kafka/image/LocalReplicaChanges.java

```diff
@@ -77,11 +77,13 @@ public Map<String, Uuid> topicIds() {
     @Override
     public String toString() {
```

Comment:
nit: Could we move `directoryIds()` method above `toString()` method for better readability?